README.md

```diff
@@ -38,8 +38,8 @@ This project has received funding from the European Union’s Horizon 2020 resea
 The RxROS2 library depends on and uses the following software:
 
  1. Ubuntu Focal Fossa 20.04 and ROS2 Foxy Fitzroy
- 1. Reactive C++ (RxCpp) v4.0
- 1. Reactive Python (RxPy) v3.1.0
+ 2. Reactive C++ (RxCpp) v4.0
+ 3. Reactive Python (RxPy) v3.1.0
```

Comment:
```suggestion
 1. Reactive C++ (RxCpp) v4.0
 1. Reactive Python (RxPy) v3.1.0
```

this is markdown. The renderer will automatically count from 1, 2 to 3.

You don't need to do that yourself.